data-raw/workflow/workflow_union_3_years_in_one_table_pseudonymised.sql

```diff
@@ -0,0 +1,63 @@
+-- To run from DALL_REF
+
+create table int646_base_20200401_20230331_pseudo nologging compress for query low as
+
+select * from
+(select RawToHex(standard_hash(nhs_no)) nhs_no_hash, base.*, opdd.lsoa_code
+from dall_ref.int646_base_20200401_20230331 base
+left join (select census_lower lsoa_code
+           from dim.ons_postcode_data_dim) opdd
+on base.bsa_postcode = opdd.postcode)
+
+pivot -- remove columns named in 'for': https://stackoverflow.com/a/33220953/8519200
+( 
+  max(1) -- fake  
+  for (bsa_postcode, bsa_sla, nhs_no) -- put the undesired columns here
```

Comment:
What is the rationale behind removing BSA-side address when matched addresses remain, many of which are exact matches?